test/dlc_tests/sanity/quick_checks/test_dlc_developer_config.py

```diff
@@ -50,14 +53,18 @@ def test_developer_config_wrappers_defaults():
     Test defaults of config file wrappers
     """
     # Check test settings
-    assert config.are_sm_efa_tests_enabled() is False
     assert config.is_sanity_test_enabled() is True
     assert config.is_sm_local_test_enabled() is False
     assert config.is_sm_remote_test_enabled() is False
+    assert config.is_sm_efa_test_enabled is False
+    assert config.is_sm_rc_test_enabled is False
     assert config.get_sagemaker_remote_efa_instance_type() == ""
     assert config.is_ecs_test_enabled() is True
     assert config.is_eks_test_enabled() is True
     assert config.is_ec2_test_enabled() is True
+    assert config.is_ec2_benchmark_test_enabled is False
+    assert config.is_sm_remote_tests_enabled
+    assert config.is_sm_benchmark_test_enabled is False
```

Comment:
Please change all of these to function calls.

Reply:
good catch, addressed